sigmf-spec.md

```diff
@@ -293,8 +294,30 @@ the `global` object:
 |`recorder`|false|string|The name of the software used to make this SigMF recording.|
 |`license`|false|string|A URL for the license document under which the recording is offered; when possible, use the canonical document provided by the license author, or, failing that, a well-known one.|
 |`hw`|false |string|A text description of the hardware used to make the recording.|
+|`geolocation`|false|GeoJSON or string|Location of the recording system.|
```

Comment:
I think we need to be SUPER clear that the only thing supported is a `GeoJSON **Point**`

Reply:
Agree. I will fix this.